cypress/pageObject/Patients/PatientLocation.ts

```diff
@@ -0,0 +1,229 @@
+export interface LocationData {
+  form?: string;
+  name: string;
+  bedsCount?: string;
+  description?: string;
+  status?: "Active" | "Inactive" | "Unknown";
+  opStatus?: string;
+}
+
+export class PatientLocation {
+  private selectors = {
+    locationForm: '[data-cy="location-form-options"]',
+    addLocationButton: '[data-cy="add-location-button"]',
+    sidebarContent: '[data-sidebar="content"]',
+    locationTab: '[data-cy="settings-locations-tab"]',
+    nameInput: '[data-cy="location-name-input"]',
+    bedCountsSelect: '[data-cy="bed-counts-select"]',
+    locationDescription: '[data-cy="location-description"]',
+    locationStatus: '[data-cy="location-status"]',
+    operationalStatus: '[data-cy="operational-status"]',
+    enableBulkCreation: '[data-cy="enable-bulk-creation-checkbox"]',
+    currentLocationBadge: '[data-cy="current-location-badge"]',
+    addLocationBadge: '[data-cy="add-encounter-location"]',
+    updateLocationButton: '[data-cy="update-encounter-location-button"]',
+    associatedLocationStatus: '[data-cy="associated-location-status"]',
+    saveStatusButton: '[data-cy="update-associated-location-status-button"]',
+    locationSearchTrigger: '[data-cy="location-search-trigger"]',
+    submitLocationAssociation:
+      '[data-cy="submit-encounter-location-association"]',
+  };
+
+  // Navigation
+  navigateToSettings() {
+    cy.get('[data-sidebar="content"]').contains("Settings").click();
```

Comment:
```suggestion
    cy.get('[data-sidebar="content"]').contains("Settings").click();
```

use cy,verifyandclick command function

Reply:
it was reused from existing PatientCreation POM code.
will make the changes

Reply:
made the change